packages/spectacular/src/lib/pipe-testing/common-pipes.integration.spec.ts

```diff
@@ -0,0 +1,131 @@
+import { CurrencyPipe, DatePipe, DecimalPipe, PercentPipe } from '@angular/common';
+
+import { SpectactularPipeFixture } from './spectacular-pipe-fixture';
+import { SpectacularPipeTestbed } from './spectacular-pipe-testbed';
+import { createCommonPipeOptions } from './test-util/create-common-pipe-options';
+
+describe('Pipe testing API', () => {
+  describe(DecimalPipe.name, () => {
+    beforeEach(() => {
+      fixture = SpectacularPipeTestbed.createPipe<number>(
+        createCommonPipeOptions({
+          template: `{{ value | number:'1.1' }}`,
+          value: 123456789,
+        })
+      );
+    });
+
+    let fixture: SpectactularPipeFixture<number>;
```

Comment:
Are you keeping the fixture wording or you will move to harness. Ex. `SpectactularPipeHarness`